views/ddox.docpage.dt

```diff
@@ -14,60 +16,79 @@ block ddox.title
 		- title = prefix ~ names.join("/") ~ " - multiple declarations";
 	
 block ddox.members
+	- bool multi_page = info.docGroups.length > 1;
 
-	- if (info.docGroups.length > 1)
-		ul
-			- foreach (dg; info.docGroups)
-				- auto itm = dg.members[0];
-				li #{itm.kindCaption} #{itm.nestedName}
+	- if (multi_page)
+		nav.page-nav
+			ul
+				- foreach (i, dg; info.docGroups)
+					- auto itm = dg.members[0];
+					li: a(href="\##{i}") #{dg.kinds.joiner("/")} #{itm.nestedName}
```

Comment:
How about using `itm.nestedName.slugify`?
A not optimized, but nicely working `slugify` method that I copy/paste around from time to time:

```d
string slugify(string target) {
    import std.regex;
    import std.string : toLower;
    static immutable slugR = ctRegex!(r"[^a-z]+", "g");
    auto ret = target.toLower.replaceAll(slugR, "-");
    if(ret[0] == '-') {
        ret = ret[1..$];
    }
    if(ret[$-1] == '-') {
        ret = ret[0..$-1];
    }
    return ret;
}
```